.github/workflows/pull_request.yml

```diff
@@ -1,24 +0,0 @@
-name: Pull Request
```

Comment:
Hold on why was this deleted?
We should still have something that builds + runs unit tests (without solana) and lints.
I may have told you to rename it to _what it does_ , i.e. "Build Lint and Unit Test" instead of when it runs (pull request), but you shouldn't just delete it.

This may be why it doesn't run